app/assets/javascripts/vote_popup.js

```diff
@@ -29,18 +29,47 @@ function VotePopupModule(showNamingsLostChangesText) {
     });
 
     $change_vote_selects.change(function (event) {
-      var _this = $(this); //references the vote being changed
-      _haveVotesChanged = true;  //votes have been changed
-      getUserAttention();
+      var _this = $(this);
+      var value = _this.val();
+      var naming_id = _this.data("id");
+      _haveVotesChanged = true;
+      _this.attr("disabled", "disabled");
+
       // If setting vote to 3.0, go through all the rest and downgrade any
       // old 3.0's to 2.0.  Only one 3.0 vote is allowed.
-      if (_this.val() == "3.0") {
-        var changedNamingId = _this.data().id;
+      if (value == "3.0") {
         $change_vote_selects.each(function () {
-          if ($(this).data().id != changedNamingId && $(this).val() == "3.0")
-            $(this).val("2.0");
+          var _this2 = $(this);
+          if (_this2.data("id") != naming_id && _this2.val() == "3.0") {
+            _this2.val("2.0");
+            _this2.attr("disabled", "disabled");
```

Comment:
I note that _this2 never gets "disabled" set to null like _this does.  Could this be a problem?